app/models/category.rb

```diff
@@ -1,6 +1,6 @@
 class Category < ApplicationRecord
   has_many :posts, dependent: :destroy
 
-  validates :title, presence: true, length: {minimum: 6, maximum: 30}
-  validates :description, length: {minimum: 30, maximum: 100}
+  validates :title, presence: true, length: {minimum: 6}
+  validates :description, length: {minimum: 30}
```

Comment:
after `{` and before `}` must have a blank space

```suggestion
  validates :title, presence: true, length: { minimum: 6 }
  validates :description, length: { minimum: 30 }
```